x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -28,13 +27,10 @@ const WAIT_FOR_DELAY_MS: number = 100;
 
 export class HeadlessChromiumDriver {
   private readonly page: Page;
-  private readonly logger: Logger;
   private readonly inspect: boolean;
 
-  constructor(page: Page, { logger, inspect }: ChromiumDriverOptions) {
+  constructor(page: Page, { inspect }: ChromiumDriverOptions) {
```

Comment:
It turns out better to not have a logger tied to the instance. Instead, calling code passes in a logger to all the driver methods.